backend/api/AuthApi/index.js

```diff
@@ -39,6 +39,19 @@ const createOauthCallbackRoute = async (oauthProviderName, code, response) => {
   if (!dbUser) {
     dbUser = await UserModel.insert.createFrom(oauthProviderName, oauthProfile);
     await NotificationModel.insert.welcome_to_memcode({ userId: dbUser.id });
+    const sgMail = require('@sendgrid/mail');
```

Comment:
1. We use `import`s.
2. Hm, did you check if it works locally? It shouldn't. Imports should be on top of the file.
3. Did you run `npm install` for @sendgrid/mail? If so, - you should commit `package.json` in this PR too.

Reply:
Yes @lakesare i used npm install sendgrid
I will commit Package.json also